src/components/radios/index.md.njk

```diff
@@ -138,6 +141,10 @@ Say ‘Select if [whatever it is]’. For example, ‘Select if you are employed
 
 Say ‘Select [whatever it is]’. For example, ‘Select the day of the week you pay your rent’.
 
+#### If it's a conditionally revealed question
+
+Say '[whatever it is] cannot be blank'. For example, 'Email address cannot be blank'.
```

Comment:
Do we need to say where the error message should display (e.g. above the input)?

Reply:
yes, in fact I think this probably needs a coded example

Reply:
A coded and visual example is really important - it's non obvious how the red error styling should appear.

Reply:
@christopherthomasdesign FYI

Reply:
NB: this is how my service handles it. With a standard error in the error summary.

<img width="679" alt="Screenshot 2022-01-20 at 12 45 13" src="https://user-images.githubusercontent.com/2204224/150372789-91bba4c6-a768-413a-bca2-bbd119bd603f.png">

Importantly we don't have error styling _outside_ of the conditional because this would be confusing if the conditional were collapsed.

Reply:
Should we say something about making sure the conditional reveal with the error is displayed as open on page load, or is that obvious? Maybe the example would be enough

Reply:
@christopherthomasdesign I'd say it explicitly.

Reply:
@calvin-lau-sig7 one challenge for these error messages is there's often two things the user can do: fix the specific error in question, or answer the parent question differently. This tends to make the errors tricky to come up with and rather bespoke.

Reply:
Sure, maybe a bit more open-ended like this? 

```suggestion
Write an appropriate [error message](/components/error-message/) for the conditionally revealed question. 
```

I've tried to include guidance around the 'parent question' but it's probably not needed if the service team has followed the guidance to only ask a conditionally revealed question if it's relevant?

> Be sure to include the key part of the previous question to help the user understand they can answer it differently.

>For example, if the user has selected a radio to be contacted by 'Email' and they've left a conditionally revealed text input to enter their 'Email address' blank, say: 'Email address cannot be blank'.

Reply:
I've added in an example here now – @calvin-lau-sig7 do you think it's in the right place?

Reply:
Yep, placement looks good to me!

Reply:
@christopherthomasdesign Awesome example, thanks!

@calvin-lau-sig7 Nice wording! Maybe we could avoid repeating 'conditionally revealed question' in the body text (as it's already in the heading 'If it's a conditionally revealed question').

Also not sure about 'appropriate' - might not be clear what we mean by it.

What about something like:

```suggestion
Include an [error message](/components/error-message/) that is clearly related to the previous question.
```

Reply:
Oh I like that! 
Another thought, wondering if 'initial question' might be a better term to use than 'previous question'?

Reply:
@calvin-lau-sig7 That's a great shout, cheers!

```suggestion
Include an [error message](/components/error-message/) that is clearly related to the initial question.
```